### DISCUSSION TITLE
Does kcert work with EKS?

### DISCUSSION THREAD

**Discussion by jmart1** [nabsul/kcert#89] [General]
I recently used kcert in two k8s clusters hosted in Digital Ocean.  It works great.  I am now setting up clusters in AWS EKS.

The only guide I was able to find on how to setup HTTPS in EKS is this blog post. https://aws.amazon.com/blogs/containers/setting-up-end-to-end-tls-encryption-on-amazon-eks-with-the-new-aws-load-balancer-controller/

It looks complicated and requires cert-manager.  

Is it possible to use kcert in an EKS cluster with the AWS ALB Controller? https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/

Is it possible to use kcert in an EKS cluster with the NGINX Ingress Controller?  https://kubernetes.github.io/ingress-nginx/
 My guess is that kcert should work with NGINX Ingress Controller in EKS because it does not rely on AWS configurations, but that's just a guess based on my experience with kcert in Digital Ocean.

**Comment by nabsul** [comment]
I have tested KCert on Digital Ocean and Azure. KCert is designed to work with ingress-nginx. It really shouldn't care which cloud it's running on as long as they're not doing any weird modifications to Kubernetes.
